### PR TITLE
Add ngsutils eta swalign

### PR DESCRIPTION
**Pull request by bogemad**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

I still haven't committed greatly to this project (or any at all really). I would appreciate someone from @bioconda/core reviewing this prior to acceptance.

Cheers,

bogemad

### CODE REVIEW

File: recipes/eta/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+package:
+  name: eta
+  version: "0.9.7"
+
+source:
+  fn: eta-0.9.7.tar.gz
+  url: https://pypi.python.org/packages/56/f4/3def7582d454d511a0b48e5eabdfd1319ccb6b5fafc810fea7dee4ca6c4d/eta-0.9.7.tar.gz
+  md5: fd6d815b2a3276ca07655c4037472576
+#  patches:
```
Comment (bgruening) [inline recipes/eta/meta.yaml]: You can remove these comments here and in your recipe.

File: recipes/eta/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  # You can also put a file called run_test.py in the recipe that will be run
+  # at test time.
+
+  # requires:
+    # Put any additional test requirements here.  For example
+    # - nose
+
+about:
+  home: http://github.com/mbreese/eta/
+  license: UNKNOWN
```
Comment (bgruening) [inline recipes/eta/meta.yaml]: Adding a licence would be good. Having no license means essentially we are not allowed to distribute the package.

File: recipes/ngsutils/build.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
+mkdir -p $PREFIX/bin
```
Comment (bgruening) [inline recipes/ngsutils/build.sh]: You could add a shebang here.

File: recipes/ngsutils/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  url: https://github.com/ngsutils/ngsutils/archive/ngsutils-0.5.9.tar.gz
+  md5: 6212d6de740c42aa2c6b5f6785465860
+
+build:
+  number: 0
+  skip: True  # [py3k]
+
+requirements:
+  build:
+    - python >=2.7.0
```
Comment (bgruening) [inline recipes/ngsutils/meta.yaml]: You can remove the `>=2.7.0`

File: recipes/ngsutils/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    - coverage
+    - swalign
+
+test:
+  commands:
+    - python -m unittest discover ngsutils
+
+about:
+  home: http://ngsutils.org
+  license: LICENSE
```
Comment (bgruening) [inline recipes/ngsutils/meta.yaml]: License?

File: recipes/swalign/build.sh [hunk 6]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+$PYTHON setup.py install 
+
+# Add more build steps here, if they are necessary.
+
+# See
```
Comment (bgruening) [inline recipes/swalign/build.sh]: Remove the comments here as well.

File: recipes/swalign/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+  url: https://pypi.python.org/packages/72/97/cb26a99ee2e1796b0ab28a955e3568607826a44999654f6880d4e00174ac/swalign-0.3.3.tar.gz
+  md5: 64c55d09775d4875bc419a08a43dc6fa
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  # noarch_python: True
+  # preserve_egg_dir: True
+  # entry_points:
```
Comment (bgruening) [inline recipes/swalign/meta.yaml]: Comments removal.

File: recipes/swalign/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+  # You can also put a file called run_test.py in the recipe that will be run
+  # at test time.
+
+  # requires:
+    # Put any additional test requirements here.  For example
+    # - nose
+
+about:
+  home: http://github.com/mbreese/swalign/
+  license: UNKNOWN
```
Comment (bgruening) [inline recipes/swalign/meta.yaml]: License :)

File: recipes/ngsutils/meta.yaml (hunk 5 above)
Comment (bogemad) [inline recipes/ngsutils/meta.yaml]: Many thanks for your improvements. 

I'm confused by this one, does the 'license:' section of the meta.yaml require a link to the license file included within the source directory or just the license name (i.e. GPL, BSD, etc.)? The "LICENSE" I've put here is supposed to be referring to the file in the source directory.

Comment (bogemad) [inline recipes/ngsutils/meta.yaml]: Actually these particular licenses require that the license files are distributed with the packages. I think a solution for this is to package the licenses with the recipe as suggested by conda-forge (http://conda-forge.readthedocs.io/en/latest/meta.html#packaging-the-license-manually).

Comment (bogemad) [inline recipes/ngsutils/meta.yaml]: I'll edit to include the license_file: and license: BSD items.

Comment (johanneskoester) [inline recipes/ngsutils/meta.yaml]: Exactly. Thanks!

**Comment by bogemad** [comment]
@bgruening and @johanneskoester, thank you both for the help.

**Pull request merged**